src/mugs.js

```diff
@@ -982,6 +982,10 @@ define([
                 lstring: 'Comment',
                 visibility: 'visible',
                 widget: widgets.multilineText,
+                setter: function (mug, attr, value) {
+                    mug.p.set(attr, value);
+                    $('.fd-question-comment').text(value);
```

Comment:
D'oh not sure what I was thinking.

https://github.com/dimagi/Vellum/pull/845/commits/7f8341908a1a1dad88962285588df0075c95d2c1

Still not 100% pleased with where I put it, but decided it was alright as we do something similar in the same file and it's the file that handles the UI layer for the inputs anyways https://github.com/dimagi/Vellum/blob/edea3c6b121a1dc4114d825265c73fe88e793aaf/src/widgets.js#L351-L355